src/utils/airtable.js

```diff
@@ -0,0 +1,91 @@
+import { stringCompare } from "./string"
+import { makeSlug } from "./slug"
+
+function getLogo(Logo, LinkedinProfile) {
+  const rawLogo = Logo || LinkedinProfile?.[0]?.data.Logo
+  const logo = rawLogo?.localFiles?.[0]?.childImageSharp?.fixed
+  return logo
+}
+
+function transformCategory(data) {
+  const {
+    id,
+    data: { Name, Count, Cover, Parent },
+  } = data
+  const parent = Parent?.[0] && transformCategory(Parent?.[0])
+  return {
+    id,
+    name: Name,
+    fullName: parent ? [parent.name, Name].join(" > ") : Name,
+    count: Count,
+    cover: Cover?.localFiles?.[0]?.childImageSharp?.fluid,
+    slug: `/categories/${makeSlug(Name)}`,
+    parent,
+  }
+}
+
+export function transformCategories(data) {
+  const categories = (data.categories?.nodes || [])
+    .map(transformCategory)
+    .sort((a, b) => stringCompare(a.name, b.name))
+
+  if (typeof window === "object") {
+    // eslint-disable-next-line no-restricted-globals
+    window.categories = categories
+    // eslint-disable-next-line no-console
+    console.log({ categories })
```

Comment:
some might say we shouldn't log in production... There is no perf cost unless devtools is open, and i like being able to quickly inspect the data as a developer.